src/_sidebar.scss

```diff
@@ -77,14 +77,14 @@ $sidebar-image-other-height: 300px;
 	margin-bottom: 10px + $sidebar-image-height + 10px + $sidebar-image-other-height + 10px;
 }
 
-// Discord button
-.side [href*="discord.gg"] {
+//Discord and Twitter Buttons
+.side [href*="discord.gg"], .side [href*="https://twitter.com"] {
```

Comment:
Should have same pattern between URLs, either include `https://` in both or neither.